frameworks/helloworld/tests/test_mono_to_multi_migrate.py

```diff
@@ -0,0 +1,67 @@
+import logging
+
+import pytest
+from tests import config
+
+import sdk_install
+import sdk_plan
+import sdk_tasks
+import sdk_upgrade
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function', autouse=True)
+def configure_package(configure_security):
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+        yield  # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.sanity
+def test_old_tasks_not_relaunched():
+    # Install hello-world with single yaml
+    sdk_install.install(config.PACKAGE_NAME,
+                        config.SERVICE_NAME,
+                        3,
+                        additional_options={"service": {"yaml": "svc"}})
+    hello_task_id = sdk_tasks.get_task_ids(config.SERVICE_NAME, 'hello')
+    # Start update plan with options that have list of yaml files to make it launch in multi service mode
+    sdk_upgrade.update_service(config.PACKAGE_NAME,
+                               config.SERVICE_NAME,
+                               additional_options={"service": {"yamls": "svc,foobar_service_name"}})
+    # Ensure the old tasks do not relaunch and new tasks are launched
```

Comment:
Could switch these checks: Wait for `foobar` to finish deployment, _then_ verify that `hello-0-server` was not relaunched.

Otherwise, we may be checking too quickly that `hello-0-server` was not relaunched -- maybe it just wasn't relaunched _yet_.